packages/zoe/test/unitTests/contracts/test-simpleExchange.js

```diff
@@ -226,26 +220,23 @@ test('simpleExchange with multiple sell offers', async t => {
       want: { Asset: moola(29) },
       exit: { onDemand: null },
     });
-    const { seat: aliceSeat3 } = await zoe.redeem(
+    const { outcome: aliceOutcome3P } = await zoe.offer(
       aliceInvite3,
       aliceBuyOrderProposal,
       { Price: aliceSimoleanPurse.withdraw(simoleans(18)) },
     );
-    const aliceOfferResult3 = aliceSeat3.addOrder();
-
-    Promise.all(aliceOfferResult1, aliceOfferResult2, aliceOfferResult3).then(
-      () => {
-        const expectedBook = {
-          changed: {},
-          buys: [[{ Asset: 29 }, { Price: 18 }]],
-          sells: [
-            [{ Price: 4 }, { Asset: 3 }],
-            [{ Price: 8 }, { Asset: 5 }],
-          ],
-        };
-        t.deepEquals(publicAPI.getBookOrders(), expectedBook);
-      },
-    );
+
+    Promise.all([aliceOutcome1P, aliceOutcome2P, aliceOutcome3P]).then(() => {
```

Comment:
Tests didn't fail because the outcome isn't used in this test

Reply:
makes sense